test/unit/cli/options-test.js

```diff
@@ -36,6 +36,8 @@ var packageJsonMock = {
   version: '1.0.0'
 }
 
+var mockWebrootLocator = simple.spy(function (arg) { return arg })
```

Comment:
I didn’t know that you can stub `simple.spy`s, too, but for coherence with other code, can you do `simple.stub()` here and in the other places please?